.github/workflows/teams-notifications.yml

```diff
@@ -12,8 +12,6 @@ jobs:
     
     steps:
       - name: Notify
-        uses: davidwengier/PostAdaptiveCard@v1.0.1
+        uses: davidwengier/PostAdaptiveCard@v1.0.2
```

Comment:
What is the purpose of 'uses' in a yml ? and why there is a specific username ?

Reply:
It's not a specific username but a reference to a repo. We're using the Github action that David created and it's hosted in https://github.com/davidwengier/PostAdaptiveCard